kbs/Cargo.toml

```diff
@@ -113,3 +113,4 @@ toml = "0.9"
 
 [build-dependencies]
 tonic-build = { workspace = true, optional = true }
+chrono = "*"
```

Comment:
we can use `chrono.workspace = true` here

Reply:
Should be fixed now.